pkg/asset/internal/templates.go

```diff
@@ -369,23 +349,21 @@ spec:
           name: dns-tcp
           protocol: TCP
       - name: healthz
-        image: gcr.io/google_containers/exechealthz:1.0
+        image: gcr.io/google_containers/exechealthz-amd64:1.1
         resources:
           limits:
             cpu: 10m
-            memory: 20Mi
+            memory: 50Mi
           requests:
             cpu: 10m
-            memory: 20Mi
+            memory: 50Mi
         args:
-        - -cmd=nslookup kubernetes.default.svc.cluster.local 127.0.0.1 >/dev/null
+        - -cmd=nslookup kubernetes.default.svc.cluster.local 127.0.0.1 >/dev/null && nslookup kubernetes.default.svc.cluster.local 127.0.0.1:10053 >/dev/null
```

Comment:
huh, odd. I'll have to go back and see where that came from.

For consistency with upstream, can you update this to more closely reflect: https://github.com/kubernetes/kubernetes/blob/release-1.3/cluster/saltbase/salt/kube-dns/skydns-rc.yaml.base

(but s/replicationController/Deployment)